libmcount/dynamic.c

```diff
@@ -244,6 +244,38 @@ struct mcount_dynamic_info * create_mdi(unsigned long baseaddr,
 	return mdi;
 }
 
+struct module_list {
+	struct list_head	list;
+	char*			module_name;
```

Comment:
Rename it to 'name' so that we can use 'module->name' later.

Reply:
it changed!